src/bio2bel/sources/rhea.py

```diff
@@ -0,0 +1,121 @@
+                ?compound rh:name ?compound_name .
+                OPTIONAL {
+                    ?compound rh:reactivePart ?part_id .
+                    ?part_id rh:chebi ?chebi .
+                    ?part_id rh:name ?reactivePart .
+                } .
+                OPTIONAL {?compound rh:chebi ?chebi} .
+                OPTIONAL {?compound rh:underlyingChebi ?chebi}
+            }
+            """
+        result = g.query(q, initBindings={'side': side_uri})
+        # Get an iterable of the compounds (no need to remove duplicates since dictionary keys must be unique)
+        compounds = map(lambda x: x[0], result)
+        # Create a dictionary that will contain the nodes associated with a compound (for GenericCompounds with multiple ReactiveParts)
+        # Goal: for compounds with multiple ReactiveParts, link those reactiveParts together so they can easily be handled later
+        #   For now, we will ignore these compounds.
+        nodes_by_compound: Dict[str, List[dsl.BaseEntity]] = {c: [] for c in compounds}
+        for r in set(result):
+            compound_name, chebi_uri, reactive_part_name = r
+            # Based on the way the OPTIONAL modifier works, the SELECT query may return ?compound_name entries by themselves, without any ?chebi information
+            if not chebi_uri:
+                continue
+            # Remove the namespace from the chebi_uri to access the identifier
+            identifier = chebi_uri.replace(CH_NAMESPACE, '')
+            # Use the compound name if no reactive part is available
+            name = reactive_part_name if reactive_part_name else compound_name
+            # Build an abundance node, and append it to the node list under the correct compound
+            node = dsl.Abundance(namespace=CHEBI, name=name, identifier=identifier)
+            nodes_by_compound[compound_name].append(node)
+
+        for compound, node_list in nodes_by_compound.items():
+            node = None
+            if len(node_list) == 1:
+                node = node_list[0]
+            else:
+                # If there are multiple nodes, there must be multiple reactiveParts
+                # We have no idea how to handle this for the moment... skip it!
+                logger.debug(
+                    f'Encountered compound {compound} with multiple ReactiveParts {node_list} ...skipped.\n\
+                    PyBEL currently has no easy way to represent this type of entity, since the groupings contain meaningful information like stoichiometry and catalysis.\n \
+                    See: https://www.rhea-db.org/rhea/24960.',
+                )
+                continue
+            # Append the node to the corresponding list in participants
+            participants[i].append(node)
+
+    return participants
+
+
+def get_bel() -> pybel.BELGraph:
+    """Get the Rhea data."""
+    # Parse the RDF file
+    g = BIO2BEL_MODULE.ensure_rdf('rhea', url=URL)
```

Comment:
Add the version to the file path so if the version gets bumped, then it will know to download new data and store in a different place

```suggestion
    g = BIO2BEL_MODULE.ensure_rdf('rhea', VERSION, url=URL)
````